pkg/services/live/channels.go

```diff
@@ -0,0 +1,49 @@
+package live
+
+import (
+	"encoding/json"
+	"math/rand"
+	"time"
+)
+
+// ChannelInfo holds metadata about each channel and is retruned on connection.
+// Eventually each plugin should control exactly what is in this structure
+type ChannelInfo struct {
+	Description string
+}
+
+type randomWalkeMessage struct {
```

Comment:
I guess it should be "randomWalkMessage"?
```suggestion
type randomWalkMessage struct {
```